### PR TITLE
Issue #KN-439 feat: Added the variable for relative_path_prefix

### PR DESCRIPTION
**Pull request by AmiableAnil**

### CODE REVIEW

File: kubernetes/helm_charts/datapipeline_jobs/values.j2 [hunk 1]
```diff
@@ -151,6 +151,7 @@ post-publish-processor:
 
     cloudstorage {
       metadata.replace_absolute_path={{ cloudstorage_replace_absolute_path | default('false') }}
+      relative_path_prefix={{ cloudstorage_relative_path_prefix_content }}
```
Comment (krgauraw) [inline kubernetes/helm_charts/datapipeline_jobs/values.j2:154]: @anilguptatarento Please add default value for the variable cloudstorage_relative_path_prefix_content

File: kubernetes/helm_charts/datapipeline_jobs/values.j2 [hunk 2]
```diff
@@ -663,6 +668,14 @@ qrcode-image-generator:
         margin=1
     }
 
+    cloudstorage {
+      metadata.replace_absolute_path={{ cloudstorage_replace_absolute_path | default('false') }}
+      relative_path_prefix={{ cloudstorage_relative_path_prefix_dial }}
```
Comment (krgauraw) [inline kubernetes/helm_charts/datapipeline_jobs/values.j2:673]: @anilguptatarento Please add default value for cloudstorage_relative_path_prefix_dial

**Review by krgauraw — CHANGES_REQUESTED** [review]

File: kubernetes/helm_charts/datapipeline_jobs/values.j2 (hunk 2 above)
Comment (AmiableAnil) [reply kubernetes/helm_charts/datapipeline_jobs/values.j2:673]: Added the default in main.yml

File: kubernetes/helm_charts/datapipeline_jobs/values.j2 (hunk 1 above)
Comment (AmiableAnil) [reply kubernetes/helm_charts/datapipeline_jobs/values.j2:154]: Added the default in main.yml

File: kubernetes/ansible/roles/flink-jobs-deploy/defaults/main.yml [hunk 3]
```diff
@@ -396,6 +396,6 @@ video_stream_topic_name: "{{ env_name }}.live.video.stream.request"
 content_hierarchy_keyspace_name: "{{ hierarchy_keyspace_name }}"
 questionset_hierarchy_keyspace_name: "{{ hierarchy_keyspace_name }}"
 
-
-cloudstorage_replace_absolute_path: 'true'
```
Comment (vinukumar-vs) [inline kubernetes/ansible/roles/flink-jobs-deploy/defaults/main.yml:400]: Why did you removed this property

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]

File: kubernetes/ansible/roles/flink-jobs-deploy/defaults/main.yml (hunk 3 above)
Comment (AmiableAnil) [reply kubernetes/ansible/roles/flink-jobs-deploy/defaults/main.yml:400]: this is moved in private repo

**Review by vinukumar-vs — APPROVED** [review]

**Review by vinukumar-vs — DISMISSED** [review]

**Pull request closed without merging**